Sources/Controllers/Map/Layers/OAWeatherContourLayer.mm

```diff
@@ -122,24 +122,40 @@ - (BOOL) updateLayer
     if ([[OAPluginsHelper getPlugin:OAWeatherPlugin.class] isEnabled])
     {
         NSString *parameterName = self.app.data.contourName;
-        OsmAnd::BandIndex band = WEATHER_BAND_UNDEFINED;
+        OsmAnd::BandIndex band = WEATHER_BAND_NOTHING;
 
         OAMapStyleParameter *tempContourLinesParam = [_styleSettings getParameter:WEATHER_TEMP_CONTOUR_LINES_ATTR];
         OAMapStyleParameter *pressureContourLinesParam = [_styleSettings getParameter:WEATHER_PRESSURE_CONTOURS_LINES_ATTR];
         OAMapStyleParameter *cloudContourLinesParam = [_styleSettings getParameter:WEATHER_CLOUD_CONTOURS_LINES_ATTR];
         OAMapStyleParameter *windContourLinesParam = [_styleSettings getParameter:WEATHER_WIND_CONTOURS_LINES_ATTR];
         OAMapStyleParameter *precipContourLinesParam = [_styleSettings getParameter:WEATHER_PRECIPITATION_CONTOURS_LINES_ATTR];
-
+        
+        BOOL bandWasChanged = NO;
```

Comment:
why `bandWasChanged` is needed? can it be replaced with code like `band != WEATHER_BAND_NOTHING`?